docs/sources/experiments/algorithm.md

```diff
@@ -110,11 +110,16 @@ a column or SQL expression representing a numeric quantity present in the `from_
 of aggregate functions we want to use. The aggregate function is applied to the quantity.
 * Each `group` is a column applied to the GROUP BY clause. Generally this is 'entity_id', but higher-level groupings
 (for instance, 'zip_code') can be used as long as they can be rolled up to 'entity_id'.
+* By default the query is joined with the cohort table (see 'state table' above) to remove unnecessary rows. If `save_all_features` is passed to the Experiment this is not done.
 
```

Comment:
Personally, save_all_features makes me think of columns and not rows... Something like 'features_ignore_cohort'. or 'features_besides_cohort', or 'features_for_all' :)

Reply:
I agree. My preference is `features_ignore_cohort`, I'll change that